woocommerce/woocommerce
# Update product data meta box checkboxes UI

### Changes proposed in this Pull Request:

Closes #50616

Performed several UI changes to the product meta box checkboxes (p6q8Tx-5qX-p2#comment-9840)

- Move the checkbox to the left
- Increase the space between items to 32px
- Use 8px between checkbox & label to align with Dotorg.
- Add `has-checkbox` to labels with checkbox

#### Before:

<img width="780" alt="image" src="https://github.com/user-attachments/assets/596fe500-e403-4008-affb-c1bc6175866f">

#### After:

<img width="851" alt="image" src="https://github.com/user-attachments/assets/2d4b1109-c6df-424b-a315-e010710a8926">

### How to test the changes in this Pull Request:

1. Go to `Products > Add new`
2. Observe the checkboxes are no positioned on the left side
3. Observe no colon character labels and checkboxes
4. Observe the distance between label and checkbox is 8px
5. Observe the distance between labels are 32px



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
